Cargo.toml

```diff
@@ -10,7 +10,7 @@ readme = "README.md"
 publish = false
 default-run = "vector"
 autobenches = false # our benchmarks are not runnable on their own either way
-rust-version = "1.66.0"
+rust-version = "1.68.0"
```

Comment:
This is meant to reflect the MSRV of Vector. We could probably add a comment to that effect since I can see why you would have updated it here. Unless the MSRV is actually 1.68.0 now?

Reply:
I didn't realize that. I'll revert and add a comment.